StripeFinancialConnections/StripeFinancialConnections/Source/Native/PartnerAuth/PartnerAuthDataSource.swift

```diff
@@ -32,6 +34,9 @@ final class PartnerAuthDataSourceImplementation: PartnerAuthDataSource {
     private let clientSecret: String
     let analyticsClient: FinancialConnectionsAnalyticsClient
     let reduceManualEntryProminenceInErrors: Bool
+    var disableAuthSessionRetrieval: Bool {
+        return manifest.features?["bank_connections_disable_defensive_auth_session_retrieval_on_complete"] == true
```

Comment:
[Grep to double-check spelling](https://livegrep.corp.stripe.com/search/stripe?q=bank_connections_disable_defensive_auth_session_retrieval_on_complete&fold_case=auto&regex=false&context=true)

Note that this should ONLY return `true` if backend has this set as a flag. NULL == enabled.

I also tested this by setting the manifest manually in init.

Reply:
[PR for feature](https://git.corp.stripe.com/stripe-internal/pay-server/pull/666331)